src/subdomains/core/history/controllers/transaction.controller.ts

```diff
@@ -80,6 +83,27 @@ export class TransactionController {
     return this.getHistoryData(query, ExportType.COMPACT, res);
   }
 
+  @Get('uid/:uid')
+  @ApiOkResponse({ type: TransactionDto })
+  async getSingleTransaction(@Param('uid') uid: string): Promise<TransactionDto> {
```

Comment:
Should we just use `@Get(':uid')` or do you want to reserve this for a future call with the "real" ID?